### DISCUSSION TITLE
Homekit support

### DISCUSSION THREAD

**Discussion by wabisabi888** [arendst/Tasmota#11033] [Q&A] [answered]
I used tuya-convert to OTA flash a smart plug with v9.2.0 of Tasmota. Browsed to it using a web browser and used Configure Other to enter the [template](https://templates.blakadder.com/stitch_27937.html), save, reboot. I can browse to it in a browser and toggle on/off etc.  How do I add this accessory to the Home app on iOS? ie. make it homekit compatible.

**Comment by Jason2866** [comment, marked as answer]
See [Homebridge](https://tasmota.github.io/docs/Homebridge/)
